kubernetes/windows/geneva/main.ps1

```diff
@@ -65,4 +63,4 @@ Start-Transcript -Path main.txt
 
 Set-EnvironmentVariables
 
-Invoke-Expression ((Join-Path $containerRoot \opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe) + " -useenv")
+Invoke-Expression ".\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe -useenv"
```

Comment:
Why change this?

Reply:
Either way works

Before main.ps1 was executing in the /opt/amalogswindows/scripts/powershell directory so I used an absolute path to avoid having "../../../"
I removed the workdir in the dockerfile so now main.ps1 is executing within the container root directory and the relative path is simpler